content/en/agent/kubernetes/daemonset_setup.md

```diff
@@ -276,20 +279,26 @@ Mount `/var/lib/docker/containers` as well, since `/var/log/pods` is symlink to
 Mount the docker socket into the Datadog Agent:
 
 ```
+  (...)
+    env:
+      - {name: DD_CRI_SOCKET_PATH, value: /host/var/run/docker.sock}
+      - {name: DOCKER_HOST, value: unix:///host/var/run/docker.sock}
   (...)
     volumeMounts:
       (...)
-      - name: dockersocket
-        mountPath: /var/run/docker.sock
+      - name: dockersocketdir
+        mountPath: /host/var/run
   (...)
   volumes:
     (...)
     - hostPath:
-        path: /var/run/docker.sock
-      name: dockersocket
+        path: /var/run
+      name: dockersocketdir
   (...)
 ```
 
+**Note**: Mounting only the `docker.sock` socket instead of the whole directory containing it prevents the agent from recovering after a docker daemon restart.
```

Comment:
```suggestion
**Note**: Mounting only the `docker.sock` socket instead of the whole directory containing it prevents the Agent from recovering after a Docker daemon restart.
```